lib/lambda_ethereum_consensus/state_transition/epoch_processing.ex

```diff
@@ -0,0 +1,33 @@
+defmodule LambdaEthereumConsensus.StateTransition.EpochProcessing do
+  @moduledoc """
+  This module contains utility functions for handling epoch processing
+  """
+
+  alias ChainSpec
+  alias SszTypes.BeaconState
+
+  @spec process_eth1_data_reset(BeaconState.t()) :: BeaconState.t()
+  def process_eth1_data_reset(state) do
+    next_epoch = get_current_epoch(state) + 1
+    epochs_per_eth1_voting_period = ChainSpec.get("EPOCHS_PER_ETH1_VOTING_PERIOD")
+
+    if rem(next_epoch, epochs_per_eth1_voting_period) == 0 do
+      %BeaconState{state | eth1_data_votes: []}
```

Comment:
you are not doing anything on this line. In Elixir, values are immutable. So you should assign the result to state

Reply:
Got it! Thanks